.gitignore

```diff
@@ -135,15 +132,37 @@ build
 # Gradle:
 .idea/**/gradle.xml
 .idea/**/libraries
-.idea
 
 # Website
 /site/Gemfile.lock
 /site/.sass-cache
 /site/_site
 
-# Pytest Cache
-**/.pytest_cache
-
-# Vscode
-.vscode/
+#added by zhanghan#
```

Comment:
Can we keep this file organized by the categories (e.g. java, python, logs, IDE, etc)?